infrastructure/terraform/modules/core/outputs.tf

```diff
@@ -3,6 +3,21 @@ output "core_id" {
 }
 
 output "loadbalancer" {
-    description = "The URL for the load balancer of the cluster. Used to access the UI via the browser"
-    value       = data.kubernetes_service.ingress.status.0.load_balancer.0.ingress.0.hostname
+  description = "The URL for the load balancer of the cluster. Used to access the UI via the browser"
+  value       = data.kubernetes_service.ingress.status.0.load_balancer.0.ingress.0.hostname
+}
+
```

Comment:
The api_host output holds also the protocol (http or https)

Reply:
I wonder if we can also print out the description of the output.. Right now it is just the title on the CLI.

Reply:
Yeah that would be good. Terraform doesn't have an option for this unfortunately. So this we can do if we control the output from the CLI.